Development/cpprest/uri_schemes.h

```diff
@@ -22,6 +23,13 @@ namespace web
 
     inline utility::string_t http_scheme(bool secure) { return secure ? uri_schemes::https : uri_schemes::http; }
     inline utility::string_t ws_scheme(bool secure) { return secure ? uri_schemes::wss : uri_schemes::ws; }
+
+    // Check if the URI is secure
+    inline bool get_secure(const web::uri& uri)
```

Comment:
I used `get_secure` because cpprestsdk websocketpp\uri.hpp using that name. But I am happy to change it to `is_secure`. btw initially I did use is_secure too until I spotted the name used in cpprestsdk.

Reply:
get_secure removed to is_secure